pkg/container/types/types.go

```diff
@@ -359,15 +359,15 @@ func (t T) FixedLength() int {
 	case T_int64, T_uint64, T_datetime, T_float64, T_timestamp:
 		return 8
 	case T_decimal64:
-		return 8
+		return -8
 	case T_decimal128:
-		return 16
+		return -16
```

Comment:
Why you want to return -8?   Isn't decimal64 8 fixed, 8 bytes?

Reply:
As an example, there are two decimal values, 1.2 and 1.2. Because they have different precision, the actual underlying values may be 1200 and 12000 respectively, so they cannot be handled directly. A special treatment is needed to do an alignment operation.